### PR TITLE
Maint/master/1.0.3 release

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
       "version_requirement": ">= 4.7.0 < 6.0.0"
     }
   ],
-  "pdk-version": "1.4.1",
-  "template-url": "file:///opt/puppetlabs/pdk/share/cache/pdk-templates.git",
-  "template-ref": "1.4.1-0-g52adbbb"
+  "pdk-version": "1.5.0",
+  "template-url": "https://github.com/puppetlabs/pdk-templates",
+  "template-ref": "heads/master-0-g34e3266"
```
Comment (jhelwig) [inline metadata.json]: Will this end up pinning the template to commit `34e3266`, or will that automatically get updated with a `pdk update`?

Comment (melissa) [reply metadata.json]: I believe it should update when I run `pdk update` next, but I'm not 100% sure about that. @bmjen ?

Comment (bmjen) [reply metadata.json]: @melissa yep, next time you run `pdk update`, PDK will pull the latest from GitHub master to perform the diff. The SHA was intended to allow us to determine whether the module was up-to-date already or not, but due to possible `.sync.yml` changes, the diff is always run, so the SHA isn't used.

**Comment by kris-bosland** [comment]
Is there a ticket I should document the merge sha on?

**Pull request merged**